R/data-pronoun.R

```diff
@@ -0,0 +1,36 @@
+#' Use the tidy eval data pronoun in your package
+#'
+#' Does setup necessary to use rlang's `.data` pronoun in your package. See [rlang::.data] for more.
+#' This function requires the use roxygen.
+#' * Adds rlang to "Imports" in `DESCRIPTION`.
+#' * Imports the `.data` object specifically, which is necessary for internal
+#'   use.
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' use_data_pronoun()
+#' }
+use_data_pronoun <- function() {
+  check_is_package("use_data_pronoun()")
+  check_uses_roxygen("use_data_pronoun()")
+
+  use_dependency("rlang", "Imports", min_version = "0.1.2")
+
+  if (has_package_doc()) {
+    roxygen_ns_append("@importFrom rlang .data") && roxygen_update()
+    return(invisible(TRUE))
+  }
```

Comment:
Can you please put the remaining code in an `else` branch? I think that makes it more clear there's two options here — either adding automatically or telling the user how to add manually.